lib/pages/home_page.dart

```diff
@@ -0,0 +1,1437 @@
+import 'package:flutter/material.dart';
+
+import '../themes/colors.dart';
+
+class HomePage extends StatelessWidget {
+  const HomePage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return ListView(
+      children: [
+        Column(
+          children: [
+            Container(
+              width: MediaQuery.of(context).size.width ,// 393,
+              height: MediaQuery.of(context).size.height, //852,
+              clipBehavior: Clip.antiAlias,
+              decoration: ShapeDecoration(
+                color:backgroundColor ,
+                shape: RoundedRectangleBorder(
+                  borderRadius: BorderRadius.circular(24),
+                ),
+              ),
+              child: Stack(
+                children: [
+                  Positioned(
+                    left: 0,
+                    top: 59,
+                    child: Container(
```

Comment:
Use sizedbox to add whitespace to layout.

There are flutter plugins to flag such issues.